<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowcertif</class>
 <widget class="QMainWindow" name="MainWindowcertif">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1211</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1081</width>
      <height>611</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">selection-background-color: qradialgradient(spread:repeat, cx:0.5, cy:0.5, radius:0.077, fx:0.5, fy:0.5, stop:0 rgba(0, 169, 255, 147), stop:0.497326 rgba(0, 0, 0, 147), stop:1 rgba(0, 169, 255, 147));</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>EDIT CERTIFICAT</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_ajouter">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>250</y>
        <width>381</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(143, 143, 143);
selection-background-color: rgb(138, 138, 138);</string>
      </property>
      <property name="text">
       <string>AJOUTER</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_idcertificat">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>50</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:20px;
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>200</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:20px;
background-color: rgb(62, 125, 188);</string>
      </property>
      <property name="text">
       <string>RESULTAT D'AUDIT</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>50</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:20px;
background-color: rgb(62, 125, 188);</string>
      </property>
      <property name="text">
       <string>ID DE CERTIFICAT</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>100</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:20px;
background-color: rgb(62, 125, 188);</string>
      </property>
      <property name="text">
       <string>OBJECTIF DE CERTIFICAT</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>100</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:20px;
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>150</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:20px;
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>150</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:20px;
background-color: rgb(62, 125, 188);</string>
      </property>
      <property name="text">
       <string>DUREE DU CERTIFICAT</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>200</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:20px;
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>-10</y>
        <width>321</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>28</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 28pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 255);</string>
      </property>
      <property name="text">
       <string>CERTIFICAT</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>0</y>
        <width>1091</width>
        <height>611</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0.0105263, y1:0.051, x2:1, y2:1, stop:0 rgba(107, 154, 255, 255), stop:1 rgba(222, 148, 222, 255));</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>420</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:20px;
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>370</y>
        <width>391</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 255, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 255);
</string>
      </property>
      <property name="text">
       <string>ENTRER L'ID DE CERTIFICAT A SUPPRIMER</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2_supprimer">
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>410</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(143, 143, 143);
selection-background-color: rgb(138, 138, 138);</string>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="PDF">
      <property name="geometry">
       <rect>
        <x>950</x>
        <y>480</y>
        <width>91</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">alternate-background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
      </property>
      <property name="text">
       <string>PDF</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../Downloads/PDFI.jpg</normaloff>../../../Downloads/PDFI.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="statistiquesButton">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>490</y>
        <width>101</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>STATS</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../Downloads/STA2.jpg</normaloff>../../../Downloads/STA2.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_display">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>490</y>
        <width>91</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>LCD</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../Downloads/LOI.png</normaloff>../../../Downloads/LOI.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>551</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>DONNER L'ID DE CERTIFICAT A MODIFIER</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="idCertificatLineEdit">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>10</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:20px;
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="modifierButton">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>80</y>
        <width>141</width>
        <height>151</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">alternate-background-color: rgb(2, 46, 91);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../Downloads/MODIF.jpg</normaloff>../../../Downloads/MODIF.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:20px;
background-color: rgb(62, 125, 188);</string>
      </property>
      <property name="text">
       <string>OBJECTIFCERIFICAT</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="objectifCertificatLineEdit">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>60</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:20px;
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:20px;
background-color: rgb(62, 125, 188);</string>
      </property>
      <property name="text">
       <string>DUREECERTIFICAT</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="dureeCertificatLineEdit">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>120</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:20px;
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>179</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:20px;
background-color: rgb(62, 125, 188);</string>
      </property>
      <property name="text">
       <string>RESULTATDAUDIT</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="resultatAuditLineEdit">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>180</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:20px;
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>611</width>
        <height>241</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:20px;
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>240</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 255);</string>
      </property>
      <property name="text">
       <string>LISTE DES CERTIFICAT</string>
      </property>
     </widget>
     <zorder>label_14</zorder>
     <zorder>pushButton_ajouter</zorder>
     <zorder>lineEdit_idcertificat</zorder>
     <zorder>label</zorder>
     <zorder>label_2</zorder>
     <zorder>label_3</zorder>
     <zorder>lineEdit_2</zorder>
     <zorder>lineEdit_3</zorder>
     <zorder>label_4</zorder>
     <zorder>lineEdit_4</zorder>
     <zorder>label_5</zorder>
     <zorder>lineEdit_5</zorder>
     <zorder>label_15</zorder>
     <zorder>pushButton_2_supprimer</zorder>
     <zorder>PDF</zorder>
     <zorder>statistiquesButton</zorder>
     <zorder>pushButton_display</zorder>
     <zorder>label_17</zorder>
     <zorder>idCertificatLineEdit</zorder>
     <zorder>modifierButton</zorder>
     <zorder>label_18</zorder>
     <zorder>objectifCertificatLineEdit</zorder>
     <zorder>label_19</zorder>
     <zorder>dureeCertificatLineEdit</zorder>
     <zorder>label_20</zorder>
     <zorder>resultatAuditLineEdit</zorder>
     <zorder>tableView</zorder>
     <zorder>label_8</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>CHERCHER</string>
     </attribute>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>20</y>
        <width>391</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>ENTRER L'ID DE CERTIFICAT A CHERCHER</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_3">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>160</y>
        <width>631</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:20px;
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cherchercertif_2">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>70</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>CHERCHER</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../Downloads/RECHER.jpg</normaloff>../../../Downloads/RECHER.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>90</width>
        <height>100</height>
       </size>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_chercher">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>70</y>
        <width>113</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:20px;
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1071</width>
        <height>581</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0.0105263, y1:0.051, x2:1, y2:1, stop:0 rgba(107, 154, 255, 255), stop:1 rgba(222, 148, 222, 255));</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>label_16</zorder>
     <zorder>label_9</zorder>
     <zorder>tableView_3</zorder>
     <zorder>cherchercertif_2</zorder>
     <zorder>lineEdit_chercher</zorder>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>trier les certificat </string>
     </attribute>
     <widget class="QTableView" name="tableView_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>140</y>
        <width>691</width>
        <height>271</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:20px;
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>-12</x>
        <y>-1</y>
        <width>1091</width>
        <height>581</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0.0105263, y1:0.051, x2:1, y2:1, stop:0 rgba(107, 154, 255, 255), stop:1 rgba(222, 148, 222, 255));</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>20</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>CERTIFICAT TRIER SELON ID</string>
      </property>
     </widget>
     <zorder>label_10</zorder>
     <zorder>tableView_2</zorder>
     <zorder>label_21</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1211</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menugestipn_des_certificats">
    <property name="title">
     <string>gestion des certificats</string>
    </property>
   </widget>
   <addaction name="menugestipn_des_certificats"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
